live/PapapiuLab/0.1.0.ini

```diff
@@ -0,0 +1,510 @@
+# Print profiles for Papapiu printers.
+# Based on the profiles from PR: https://github.com/slic3r/slic3r-profiles/pull/13 by top-gun.
+# The graphics for the bed (SVG) was created by PwnCNC and published on Thingiverse under the Creative Commons Attribution license: https://www.thingiverse.com/thing:4861532/files
+
+[vendor]
+# Vendor name will be shown by the Config Wizard.
+name = Papapiu
+config_version = 0.1.0
+config_update_url = https://files.prusa3d.com/wp-content/uploads/repository/PrusaSlicer-settings-master/live/PapapiuLab/
+
+# The printer models will be shown by the Configuration Wizard in this order,
+# also the first model installed & the first nozzle installed will be activated after install.
+# Printer model name will be shown by the installation wizard.
+
+[printer_model:Papapiu_N1S]
+name = Papapiu N1S
+variants = 0.4
+technology = FFF
+bed_model = N1_Bed.stl
+bed_texture = N1_Texture.png
```

Comment:
You have to either also add `thumbnail = N1_thumbnail.png` here or rename the thumbnail file to `Papapiu_N1S_thumbnail.png`.